includes/class-wc-payments-account.php

```diff
@@ -182,8 +182,12 @@ public function get_statement_descriptor() {
 	 * @throws API_Exception Bubbles up from get_cached_account_data.
```

Comment:
Should we remove this comment now that an API_Exception won't bubble up?

Reply:
Hm, that comment makes me think whether we should catch this or not. And quick search revealed some other places where `get_cached_account_data` not wrapped 🤔

@luizreis how do you think, should we wrap them as well? If so, could you please finish it because I'm at the end of shift today and AFK tomorrow?

Reply:
I think the issue with this call is that it's used when localizing a script, so even if the gateway is not initialized properly it would throw the error and crash the entire site. Do you think it makes sense to catch it at the moment it's used instead of inside the function?

Reply:
That's a fair point. I have a solution which will fix #551 as well :) Will ship today.